packages/frontend/remix/src/components/common/PlatCell/PlatDateFormatter.test.ts

```diff
@@ -0,0 +1,104 @@
+import { describe, expect, test } from "vitest";
+
+import { PlatDateFormatter } from "./PlatDateFormatter";
+
+describe("相対時間の計算 PlatDateFormatter", () => {
+  test("不正な引数：タイムスタンプが現在時刻を超過", () => {
+    expect(() => {
+      PlatDateFormatter({
+        platTimestamp: new Date("2021-01-30T00:00:55+09:00"),
+        currentTime: new Date("2021-01-30T00:00:54+09:00"),
+      });
+    }).toThrow("Invalid time difference: platTimestamp is later than currentTime.");
+  });
```

Comment:
このケースですが、ブラウザ側時刻がずれていた場合は通常運用で起きうるので、異常系ではなく準正常系として捌いてください。

twitter だと「現在」表示になるようなので一旦それでお願いします

```suggestion
  test("不正な引数：タイムスタンプが現在時刻を超過", () => {
    expect(() => {
      PlatDateFormatter({
        platTimestamp: new Date("2021-01-30T00:00:55+09:00"),
        currentTime: new Date("2021-01-30T00:00:54+09:00"),
      });
    }).toBe("現在");
  });
```